protobuf/mayastor.proto

```diff
@@ -500,6 +501,26 @@ message RebuildStatsReply {
   uint64 tasks_active = 7; // number of current active tasks
 }
 
+message RebuildHistoryRequest {
```

Comment:
Probably want to add this to both v1 and v0? Or at least v1

Reply:
As I figured, this file sits in version agnostic place. How do we make that exposed to various versions?

Reply:
I'm not sure how to do that since the package would be different, maybe @Abhinandan-Purkait knows?

Reply:
@dsharma-dc Even though this file sits in a version agnostic place, `mayastor.proto` refers to the v0 api only. I think there are two ways you can do it. You can have a common that is shared with v0 and v1 and that file sits here(this would need changes at other places where we compile the protobuf). Or you can just copy the same to v1/nexus ?

Reply:
Okay, Seems like you have done the second way. I arrived late here :-P